### ISSUE TITLE
Keras guide webpage: tf.keras.version links to a dead webpage.

### ISSUE THREAD

**Issue by m3rc3n4ry** [tensorflow/tensorflow#20913]
## System Information
**Have I written custom code:** No
**OS Platform and Distribution:** Windows 10 Pro
**TensorFlow installed from:** source
**TensorFlow version:** N/A
**Bazel version:** N/A
**CUDA/cuDNN version:** N/A
**GPU model and memory:** Nvidia GeForce GTX 950m 4GB
**Exact command to reproduce:** N/A

## Describe the problem
- On [this](https://www.tensorflow.org/guide/keras) TensorFlow guide website, I found that it mentions using `tf.keras.version` to check the keras version. However, it should be `tf.keras.__version__` instead of `tf.keras.version`. 

- Also, it links to a [webpage](https://www.tensorflow.org/api_docs/python/tf/keras/__version__) which throws `404` error.

![tf keras issue](https://user-images.githubusercontent.com/29359259/42868356-bb35fbf0-8a8f-11e8-81e5-50b537942d62.PNG)

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by m3rc3n4ry** [comment]
Sorry. Fixed it.

**Comment by drpngx** [comment]
Thanks!

**Comment by m3rc3n4ry** [comment]
I'm sorry for the confusion, but I said 'fixed it' to tell that I fixed the thing that the bot was talking about regarding the template. Not because the issue was fixed.

**Comment by drpngx** [comment]
I think that makes sense. Please send a PR.

**Comment by m3rc3n4ry** [comment]
I can't find the website code in tensorflow/tensorflow repository. Can you point me to it?

**Comment by drpngx** [comment]
It's probably [here](https://www.github.com/tensorflow/tensorflow/blob/master/tensorflow/docs_src/guide/keras.md#L36).

**Comment by m3rc3n4ry** [comment]
I checked that file already. There's no issue in the file, it's all correct. But the webpage is different (as shown in the screenshot above) than this Keras.md file that you shared. The **webpage** documentation has the issue.

Just like there's a tfjs-website repository for tensorflow.js website, I want to know how I can fix the issue on tensorflow.org website.

I hope this makes it clear.

**Comment by drpngx** [comment]
The file seems to say `tf.keras.__version__` at line 35, right?

**Comment by drpngx** [comment]
For the website, probably @MarkDaoust can fix it directly.

**Comment by MarkDaoust** [comment]
Yes. Thanks.

We're working on breaking the website out of the main repo so that we're not tied to release branches, as we currently are.

That will let us fix things like this more easily.

The 404 ... probably needs a fix to the api-reference generator. simple constants don't have api pages. That will take some work.

**Comment by tensorflowbutler** [comment]
Please remove the assignee, as this issue is inviting external contributions. Otherwise, remove the `contributions welcome` label. Thank you.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @MarkDaoust: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
